cmd/erasure-server-pool.go

```diff
@@ -1125,6 +1125,7 @@ func (z *erasureServerPools) CopyObject(ctx context.Context, srcBucket, srcObjec
 		// Destination is not versioned and source version ID is empty
 		// perform an in-place update.
 		if !dstOpts.Versioned && srcOpts.VersionID == "" {
+			srcInfo.Reader.Close() // We are not interested in the reader stream at this point close it.
```

Comment:
isn't it is true for all the code that calls CopyObject() underneath?

Reply:
yes, but we need to do this in erasure set layer